library/src/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -647,6 +746,21 @@ public void highlightDates(Collection<Date> dates) {
     setAdapter(adapter);
   }
 
+  /**
+   * Unhighlights all previously highlighted days.
+   * Also clears {@link #highlightedCells} and {@link #highlightedCals}.
+   */
+  public void unHighlightAllDates() {
```

Comment:
Actually, we don't use the name highlighted anywhere. Use `clearSelection()` maybe.